### PR TITLE
Fix Enum.uniq_by example

### PR DESCRIPTION
**Pull request by eksperimental**
Fix example and use iex> so it can be tested by doctest

### CODE REVIEW

File: lib/elixir/lib/enum.ex [hunk 1]
```diff
@@ -2301,8 +2301,8 @@ defmodule Enum do
       iex> Enum.uniq_by([{1, :x}, {2, :y}, {1, :z}], fn {x, _} -> x end)
       [{1, :x}, {2, :y}]
 
-      Enum.uniq_by([{a: {tea: 2}}, {b: {tea: 2}}, {c, {coffe: 1}}], fn {x, _} -> x end)
-      [a: [tea: 2], b: [tea: 2]]
+      iex> Enum.uniq_by([a: {:tea, 2}, b: {:tea, 2}, c: {:coffe, 1}],  fn {_, y} -> y end)
```
Comment (doomspork) [inline lib/elixir/lib/enum.ex]: This should be `:coffee`, shouldn't it?  Looks like there was a typo in the original.

Comment (eksperimental) [inline lib/elixir/lib/enum.ex]: thanks @doomspork ! fixed

**Comment by lexmag** [comment]
Nice. :yellow_heart:

**Comment by eksperimental** [comment]
;-)

**Pull request merged**